data-prepper-plugins/date-processor/README.md

```diff
@@ -0,0 +1,115 @@
+# Date Processor
+
+Date processor is used for adding default timestamp to event and for parsing date fields and using it as Data Prepper event timestamp.
+
+## Basic Usage
+To get started with date processor, create the following `pipeline.yaml`.
+
+```yaml
+  source:
+    file:
+      path: "/full/path/to/logs_json.log"
+      record_type: "event"
+      format: "json"
+  prepper:
+    - grok:
+        match: 
+          log: ['%{IPORHOST:clientip} \[%{HTTPDATE:timestamp}\] %{NUMBER:response_status:int}']
+          
+    - date:
+        match:
+            - key: timestamp
+              patterns: ["dd/MMM/yyyy:HH:mm:ss XX"] 
+        destination: "@timestamp"
+        timezone: "America/Chicago"
+        locale: "en_US"
+  sink:
+    - stdout:
+```
+Create the following file named `logs_json.log` and replace the `path` in the file source of your `pipeline.yaml` with the path of this file.
```

Comment:
Where is this file `logs_json.log` placed?

Reply:
This is a sample log file created by users and they can provide the path of the file in file source.